pkg/tests/ossm/prometheus_scoped_scraping_test.go

```diff
@@ -0,0 +1,134 @@
+package ossm
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+
+	"github.com/maistra/maistra-test-tool/pkg/util/check/common"
+	"github.com/maistra/maistra-test-tool/pkg/util/check/require"
+	"github.com/maistra/maistra-test-tool/pkg/util/env"
+	"github.com/maistra/maistra-test-tool/pkg/util/hack"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/retry"
+	"github.com/maistra/maistra-test-tool/pkg/util/shell"
+	"github.com/maistra/maistra-test-tool/pkg/util/test"
+)
+
+func TestOperatorCanUpdatePrometheusConfigMap(t *testing.T) {
+	test.NewTest(t).Groups(test.Full).Run(func(t test.TestHelper) {
+		hack.DisableLogrusForThisTest(t)
+
+		t.Log("This test checks if the operator can update Prometheus ConfigMap when the SMMR is updated")
+
+		meshNamespace := env.GetDefaultMeshNamespace()
```

Comment:
You directly get this from setup in ossm package

Reply:
I was trying to follow the [clusterwide_mode_test](https://github.com/maistra/maistra-test-tool/blob/da46ddcec785ab546474e5c3a7bf5df0cc052a46/pkg/tests/ossm/operator/clusterwide_mode_test.go#L28). How should I get the mesh namespace?

Reply:
meshNamespace = env.GetDefaultMeshNamespace() is the same :). I always use the variable because is defined previously in setup.go